### PR TITLE
59503 schedulable param required

### PR DESCRIPTION
**Pull request by cferris32**
## Summary

- This change is to reintroduce the 'schedulable' param as required on the get_facilities calls in vets-api, original issue/changes linked below

## Related issue(s)
- department-of-veterans-affairs/va.gov-team#59503

## Testing done

- Tested that param is required and regression tested other requests

### CODE REVIEW

**Review by aherzberg — APPROVED** [review]
Mobile spec updates LGTM.

**Review by simiadebowale — APPROVED** [review]
LGTM

**Review by kpethtel — DISMISSED** [review]
Mobile changes are good. 👍 
FYI, I've already created a ticket on our side for fixing our specs later.

File: modules/vaos/app/controllers/vaos/v2/facilities_controller.rb [hunk 1]
```diff
@@ -43,6 +44,20 @@ def children
       def type
         params[:type]
       end
+
+      def schedulable
+        if Flipper.enabled?('va_online_scheduling_required_schedulable_param')
+          schedulable = params.require(:schedulable)
+          if schedulable.is_a?(TrueClass) || schedulable.is_a?(FalseClass) ||
```
[the remark this answers is not in the document]
Comment (cferris32) [reply modules/vaos/app/controllers/vaos/v2/facilities_controller.rb]: Good catch, thanks for the idea - I've cleaned up this method

**Review by kpethtel — APPROVED** [review]
👍

**Review by simiadebowale — APPROVED** [review]

**Review by rmtolmach — APPROVED** [review]

**Pull request merged**